### PR TITLE
Feature/hot reload

### PR DESCRIPTION
**Pull request by mr-mig**
Components and styles are hot-reloaded.

I have added `./clean-dev.sh` and `./dev.sh` scripts for convenience.  
The useage is the same as in `homescreen` project:
1. If you check out a fresh master, you run `./clean_dev.sh`
2. You run `./dev.sh` anytime you wanna develop the project - it will run hotreload and backend server automatically.

### CODE REVIEW

File: app/com/yetu/controlcenter/views/index.scala.html [hunk 1]
```diff
@@ -7,16 +8,22 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>yetu Control Center</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+    <link rel="icon" type="image/ico" href="favicon.ico"/>
+  </head>
+  <body>
+  @if(Play.current.configuration.getString("yetu.frontendBundlePath").isDefined) {
```
Comment (mr-mig) [inline app/com/yetu/controlcenter/views/index.scala.html]: Note this stuff.
@jschaul @enginyoyen Any ideas how to simplify/change it?

**Comment by mr-mig** [comment]
@DaQuirm @dev-tim Have a close look at the changes - it may be useful to know those configuration details.
@larsblumberg I've added the scripts, have a close look plz.

**Comment by dev-tim** [comment]
:+1:

**Comment by larsblumberg** [comment]
LGTM now

**Pull request merged**